hack/update-go-format.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${REPO_ROOT}"/hack/util.sh
+
+util::verify_go_version
+
+cd "${REPO_ROOT}"
+find_files() {
+  find . -not \( \
+      \( \
+        -wholename './bin' \
+        -o -wholename './.git' \
+        -o -wholename './site/*' \
+        -o -wholename '*/vendor/*' \
+      \) -prune \
+    \) -name '*.go'
+}
```

Comment:
The go files only exist under `cmd` and `pkg`, so maybe we should just find these two directories, and change them together in `verify-go-format.sh`